### PR TITLE
Add the role management commands to CLI docs

### PR DESCRIPTION
**Pull request by matt-royal**
Also add a link to the existing metadata section, which was accidentally omitted.

### CODE REVIEW

**Comment by HenryBorys** [comment]
Hi @emalm, waiting on your review for this one. Thanks.

**Comment by HenryBorys** [comment]
@matt-royal What versions does this apply to? Just main?

**Comment by matt-royal** [comment]
Hi @HenryBorys. This PR applies to both main and 1.2. I believe that's true for all of the PRs to main, since there is no 1-2 docs branch and main hasn't diverged much from 1.2 yet.

**Comment by HenryBorys** [comment]
@matt-royal Yep, main and 1.2 are the same for now. What I'm wondering is whether the PR also applies to previous versions, which is often the case. It's helpful, even if it doesn't, to note as much in the PR so I don't wonder and ask. Thanks. BTW, do we need @emalm to review before merging?

File: supported-cf-cli-commands.md [hunk 1]
```diff
@@ -244,3 +248,17 @@ This section describes the metadata operations that Application Service Adapter
 | `cf labels` | Y | The `app`, `org`, and `space` resources are supported. |
 | `cf set-label` | Y | The `app`, `org`, and `space` resources are supported. |
 | `cf unset-label` | Y | The `app`, `org`, and `space` resources are supported. |
+
+## <a id="role-management"></a> User Role Management
+
+This section describes the operations for adding and removing roles that Application Service Adapter supports.
```
Comment (julian-hj) [inline supported-cf-cli-commands.md:254]: Are we currently supporting any roles other than "SpaceDeveloper"? If not, it seems a bit odd to claim that we support `set-org-role`

**Review by julian-hj — CHANGES_REQUESTED** [review]

File: supported-cf-cli-commands.md [hunk 2]
```diff
@@ -244,3 +248,17 @@ This section describes the metadata operations that Application Service Adapter
...
+These commands work for both Kubernetes users and Kubernetes service accounts. When dealing with service accounts,
```
[the remark this answers is not in the document]
Comment (tcdowney) [reply supported-cf-cli-commands.md]: Addressed in https://github.com/pivotal/docs-tas-adapter/pull/94/commits/b1953157f830be253fd3e60749dfc0c5aa1fcaf1

**Review by julian-hj — APPROVED** [review]

**Review by tcdowney — APPROVED** [review]

**Pull request merged**